x-pack/plugins/features/server/oss_features.test.ts

```diff
@@ -5,7 +5,8 @@
  */
 
 import { buildOSSFeatures } from './oss_features';
-import { featurePrivilegeIterator } from '../../security/server/authorization';
+// @ts-expect-error
+import { featurePrivilegeIterator } from './feature_privilege_iterator';
```

Comment:
workaround to remove the circular dependency between features and security. @legrego wouldn't it be more appropriate to move this privilege assertion into the `security` plugin test? https://github.com/elastic/kibana/pull/87294/files#diff-bb6d1fa1f497224ebef5f42ca080dc5baadd274b96fd8ebc99652071fd5a4a77R64

Reply:
Strictly speaking, yes! There was a reason we did it this way originally, but I'm struggling to remember why. It may have involved not wanting to export `buildOSSFeatures` from the `features` plugin, but looking at it now, it's no different than exporting `featurePrivilegeIterator` from the security plugin 🤷.

I can add a task to address this in followup